board/spi_flasher.h

```diff
@@ -181,9 +181,9 @@ void bl_can_send(uint8_t *odat) {
 void CAN1_RX0_IRQHandler(void) {
   while (CAN->RF0R & CAN_RF0R_FMP0) {
     if ((CAN->sFIFOMailBox[0].RIR>>21) == CAN_BL_INPUT) {
-      uint8_t dat[8];
-      ((uint32_t*)dat)[0] = CAN->sFIFOMailBox[0].RDLR;
-      ((uint32_t*)dat)[1] = CAN->sFIFOMailBox[0].RDHR;
+      uint32_t dat[2];
```

Comment:
BUG! See line 205

Reply:
gh, misundersood what `((uint32_t*)dat)[0] = CAN->sFIFOMailBox[0].RDLR;` does. Should be ok now.